Modules/socketmodule.c

```diff
@@ -8616,6 +8635,9 @@ socket_exec(PyObject *m)
 #ifdef  TCP_CONNECTION_INFO
     ADD_INT_MACRO(m, TCP_CONNECTION_INFO);
 #endif
+#ifdef  SIO_TCP_SET_ACK_FREQUENCY
+#define TCP_QUICKACK SIO_TCP_SET_ACK_FREQUENCY
```

Comment:
I would prefer to use ADD_INT_CONST() instead of defining the TCP_QUICKACK macro here.

Reply:
The only purpose is to activate the original #ifdef which is two lines below and adds an int macro to enable the property TCP_QUICKACK to be defined on the socket.